frontend/pages/review.tsx

```diff
@@ -35,29 +37,86 @@ function ErrorWrapper({ message }: { message: string | undefined }) {
 }
 
 function ApprovalList({ category, filter }: { category: ApprovalCategory; filter: ApprovalFilterType }) {
+  const { approvals, isApprovalsLoading, isApprovalsError, mutateApprovals } = useListApprovals(category, filter)
+
+  console.log(approvals)
+  const error = MultipleErrorWrapper(
+    `Unable to load approval page`,
+    {
+      isApprovalsError,
+    },
+    ErrorWrapper
+  )
+  if (error) return error
+
+  const getUploadCategory = (approvalCategory: ApprovalCategory) =>
+    approvalCategory === 'Upload' ? 'models' : 'deployments'
+
+  if (isApprovalsLoading || !approvals) {
+    return <Paper sx={{ mt: 2, mb: 2 }} />
+  }
+  return (
+    <Paper sx={{ mt: 2, mb: 2, pb: 2 }}>
+      <Typography sx={{ p: 3 }} variant='h4'>
+        {category === 'Upload' ? 'Models' : 'Deployments'}
+      </Typography>
+      {approvals.map((approval: any) => (
```

Comment:
This `any` isn't necessary, it should automatically be picking up the `Approval` type from `useListApprovals`